scripts/st2bootstrap-deb.template.sh

```diff
@@ -154,7 +154,18 @@ install_st2_dependencies() {
   fi
 
   sudo apt-get install -y curl
+  
+  # Various other dependencies needed by st2 and installer script
+  sudo apt-get install -y crudini
+}
+
+install_rabbitmq() {
+  # install RabbitMQ
   sudo apt-get install -y rabbitmq-server
+  sudo rabbitmqctl add_user stanley "${ST2_RABBITMQ_PASSWORD}"
```

Comment:
Can you change the username from `stanley` to `stackstorm`?  This is to be consistent with the username that we used for MongoDB.